tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -124,34 +169,72 @@ def _BDD100K_loader(path: str, dataset_type: str) -> Dataset:
 
 
 def _load_segment(dataset: Dataset, root_path: str, dataset_type: str) -> None:
-    images_directory = os.path.join(root_path, "images", dataset_type)
     labels_directory = os.path.join(root_path, "labels")
 
-    get_data = _get_data_100k if dataset_type == "100k" else _get_data_10k
     read_label_file = _read_label_file_100k if dataset_type == "100k" else _read_label_file_10k
 
     for segment_name in _SEGMENT_NAMES:
         segment = dataset.create_segment(segment_name)
-        image_paths = _utility.glob(os.path.join(images_directory, segment_name, "*.jpg"))
+        image_paths = _utility.glob(
+            os.path.join(root_path, "images", dataset_type, segment_name, "*.jpg")
+        )
 
         print(f"Reading data to segment '{segment_name}'...")
         if segment_name == "test":
             for image_path in image_paths:
                 segment.append(Data(image_path))
         else:
+            if dataset_type == "10k":
```

Comment:
`elif`?

Reply:
the codes below `if` section should run whether or not `if` runs